src/pydantic_avro/base.py

```diff
@@ -7,9 +7,9 @@ class AvroBase(BaseModel):
     """This is base pydantic class that will add some methods"""
 
     @classmethod
-    def avro_schema(cls) -> dict:
+    def avro_schema(cls, by_alias: bool = True) -> dict:
         """Return the avro schema for the pydantic class"""
```

Comment:
Can this be added to the doc string?

Reply:
Please check that the format of docstrings I used is correct. I used the same as pydantic is using. But looks like fixed.

Reply:
That all fine :)